file-share-service-desktop-client/Modules/Auth/AuthenticateView.xaml

```diff
@@ -16,33 +25,44 @@
             <ColumnDefinition Width="*" />
         </Grid.ColumnDefinitions>
 
-        <StackPanel
-            VerticalAlignment="Center"
+        <StackPanel HorizontalAlignment="Left" VerticalAlignment="Top">
+            <Image Stretch="UniformToFill" StretchDirection="Both" Source="/Modules/Auth/image/resources/UKHO-1400x240.jpg" />
+        </StackPanel>
+        <StackPanel Margin=" 0 30"
+            VerticalAlignment="Bottom"
             HorizontalAlignment="Center">
             <TextBlock FontSize="28"
                        TextWrapping="Wrap"
-                       MaxWidth="700"
-                       Margin="0 15">
+                       MaxWidth="1000"
+                       Margin="0 5">
                 The ADMIRALTY File Share Service allows you to search and download files.
             </TextBlock>
-            <Image Source="../../Views/resized_0005_Web.jpg"></Image>
```

Comment:
Could we make sure we delete the original file as well please (`../../Views/resized_0005_Web.jpg`)

Reply:
Done the required changes Kindly recheck